plugin.json

```diff
@@ -59,6 +59,44 @@
                 "display_name": "Email Settings",
                 "type": "longtext",
                 "help_text": "The JSON EmailSettings section for Mattermost."
+            },
+            {
+                "key": "ClusterWebhookAlertsEnable",
+                "display_name": "Enable Cluster Webhook Alerts",
+                "type": "bool",
+                "help_text": "Enable or disable the plugin from sending alerts to the channel defined below when cluster webhooks are received from the provisioner.",
+                "default": false
+            },
+            {
+                "key": "ClusterWebhookAlertsTeam",
+                "display_name": "Cluster Webhook Alerts Team",
+                "type": "text",
+                "help_text": "The team to send cluster webhook alerts to when enabled. This team must exist for alerts to be sent."
+            },
```

Comment:
Not a big deal but we could drop this config setting for team and just have users specify a channel ID (which is unique). That way we only have one setting for channel

Reply:
Makes sense. I will throw an update up to address this.